ruma-api/Cargo.toml

```diff
@@ -31,3 +31,7 @@ thiserror = "1.0.23"
 [dev-dependencies]
 ruma-events = { version = "=0.22.0-alpha.2", path = "../ruma-events" }
 trybuild = "1.0.38"
+
+[features]
```

Comment:
I initially skipped these features on ruma-api, but that caused the tests to fail because they include calls to the macro. This was mostly just a workaround for those failing tests.

Reply:
I see. Can you add a comment above then to clarify this, and remove the feature re-exports from `ruma`? Can decide what to do with those tests after this PR.

Reply:
Yep, sounds good.